pepper-apis/src/appengine/Housekeeping.yaml

```diff
@@ -0,0 +1,14 @@
+service: housekeeping
+runtime: java11
+instance_class: B2
+# B2 allows background threads and we can use manual scaling
+manual_scaling:
+  instances: 1
+
```

Comment:
Again, we might want to tweak these for prod environment (or even in test/staging).  We shall see.